### PR TITLE
More consistency and standardization improvements.

### PR DESCRIPTION
**Pull request by jonparrott**
* Adding argparse/docstring to cloud storage and monitoring samples.
* Updating readmes for BigQuery, Monitoring, Cloud Storage, and the Blog.
* Removing all non-appengine requirements.txt files and consolidating into a single top-level requirements.txt. This makes it easier for us to verify that the samples work with the latest version of all dependent libraries.
* Moved test dependences out of tox.ini and into requirements-dev.txt, as per standard practice.

### CODE REVIEW

File: monitoring/api/auth.py [hunk 1]
```diff
@@ -53,42 +35,39 @@
...
 
 def main(project_name):
-    # Create and return the CloudMonitoring service object.
-    service = build('cloudmonitoring', 'v2beta2',
-                    credentials=GoogleCredentials.get_application_default())
+    credentials = GoogleCredentials.get_application_default()
+    service = build('cloudmonitoring', 'v2beta2', credentials=credentials)
 
-    ListTimeseries(project_name, service)
+    list_timeseries(project_name, service)
```
Comment (tmatsuo) [inline monitoring/api/auth.py]: Apparently you need to swap project_name and service here.

**Comment by tmatsuo** [comment]
LGTM except for the simple fix

File: monitoring/api/auth.py (hunk 1 above)
Comment (jonparrott) [inline monitoring/api/auth.py]: Good catch, will do.

**Comment by jonparrott** [comment]
Fixed, merging.

**Pull request merged**